app/src/main/java/com/example/bookwormadventuresdeluxe2/Book.java

```diff
@@ -6,27 +6,34 @@
 import androidx.core.content.res.ResourcesCompat;
 
 import com.example.bookwormadventuresdeluxe2.Utilities.Status;
-import com.example.bookwormadventuresdeluxe2.R;
 
-public class Book
+import java.io.Serializable;
+
+public class Book implements Serializable
 {
     // Basic attributes for now, rest added as needed
     private String title;
     private String author;
     private String isbn;
     private String description;
     private Status status;
-
-
+    
     public Book(String title, String author, String description, String isbn, Status status)
     {
         this.title = title;
         this.author = author;
         this.description = description;
         this.isbn = isbn;
+        this.description = description;
         this.status = status;
     }
 
+    public static boolean fieldsValid(String title, String author, String description, String isbn)

```

Comment:
What do you mean by fields valid?

Reply:
It's for checking if the fields inputted for the Book object are valid. We should make this check right before instantiating a Book Object.

I'll add a javadoc to make it more clear.

Reply:
Oh I meant more like what constitutes a valid book?

Reply:
Yep